tests/test_cases_with_valid_exts.py

```diff
@@ -18,14 +19,12 @@ def test_valid_file_with_valid_defaults(self):
 
         myst_reader = MySTReader(settings)
 
-        source_path = os.path.join(TEST_CONTENT_PATH, "valid_content.md")
+        source_path = TEST_CONTENT_PATH / "valid_content.md"
         output, metadata = myst_reader.read(source_path)
+        expected = (TEST_CONTENT_PATH / "valid_content.html").read_text().strip()
 
         self.assertEqual(
-            (
-                "\n<p>This is some valid content that should pass."
-                " If it does not pass we will know something is wrong.</p>\n"
-            ),
+            expected,
             output,
         )
```

Comment:
@paugier I have only fixed this test in this test case. The remainder needs to be fixed. You can also move the expected output into `test_content/*.html` file, as I have done here.

Reply:
I guess we need to be able to modify Sphinx configuration (see https://github.com/executablebooks/MyST-Parser/issues/619).

Reply:
Yes. How MyST-parser works with the docutils API is really unclear from their docs. I think it has something to do with one of the following classes

```py
from myst_parser.sphinx_ import Parser as SphinxParser  # used in this code, but only default initialization
from myst_parser.mdit_to_docutils.sphinx_ import SphinxRenderer  # not used
```

https://myst-parser.readthedocs.io/en/latest/api/reference.html